src/lib/Microsoft.Fx.Portability.Reports.DGML/ReferenceNode.cs

```diff
@@ -105,6 +105,12 @@ public bool TryGetAPICountFromReferences(int target, out int availAPIs, out int
 
             foreach (var item in Nodes)
             {
+                if (item.UsageData == null)
```

Comment:
I feel like I've reviewed this or similar changes before. I'm concerned that this check and continue will prevent recursion (and therefore proper counting) if a Node without usage data, but containing Nodes is encountered. Ideally, the UsageData check should be around the initial calculation of avail/unavailAPIs, but still allow recursion. I'll see if I can propose a change.

Reply:
At the top...
```suggestion
                if (UsageData is null)
                {
                    availAPIs = 0;
                    unavailAPIs = 0
                }
                else
                {
                    availAPIs = UsageData[target].GetAvailableAPICalls();
                    unavailAPIs = UsageData[target].GetUnavailableAPICalls();
                }
```

Reply:
Bah, silly github. That's not the diff I want. This block should go away, and the top of the method should look like the green above.